translations: init japanese translation

nyawox opened the pull request:
Still a work in progress, but I decided to open a draft PR just so I can share the progress and hear everyone's feedback!

nyawox committed:
translations: add a japanese translation

mrshmllow reviewed (changes requested):
Please add a `src/pages/ja/index.astro`, copy the [en one](https://github.com/auxolotl/website/blob/e447a211242cb0ae55ae9cb7e97c14cae533ba57/src/pages/en/index.astro) for now.

nyawox commented:
Ready for review, but I don't consider it nearly as complete or perfect.
- I read ad-hoc as "interim", correct me if its intention was "temporary"
- I left both organization and alignment in katakana because I wasn't certain about their specific intention, but it will do fine ig
- I struggled to find a concise translation for "the work they have committed to will not change" in Japanese, so I changed to "the contribution will not be wasted."
- I'm not sure if 教育 is the appropriate term for "education" in various contexts.
- Am I correct in understanding that `used too heavily to remove or change` means `used too heavily to remove or replace`?
- Changed `Due to there being necessary bootstrapping steps for each` to `Because we require bootstrapping`
- what exactly "logistics tasks" means, the management of the details of an operation?

I'm not proffesional by any means, just an average native speaker, so feel free to nitpick

nyawox commented:
Pinging because I believe @AngryAnt is the only one who can even read basic Japanese:)

mrshmllow commented on src/pages/ja/index.astro:
```diff
@@ -0,0 +1,25 @@
+---
+import Layout from "../layouts/Layout.astro";
+import Header from "../components/Header.astro";
+import Hero from "../components/Hero.astro";
+import Values from "../components/Values.astro";
+import Goals from "../components/Goals.astro";
+import Roadmap from "../components/Roadmap.astro";
+---
+
+<Layout title="aux.computer">
+	<main>
+		<Header />
+
+		<div class="grid place-items-center">
+			<Hero />
+			<div class="prose prose-invert py-16 px-4 max-w-3xl">
+				<Values />
+				<Goals />
+				<Roadmap />
+			</div>
+		</div>
+	</main>
+</Layout>
+
+<style></style>
```
```suggestion
---
import Layout from "../../layouts/Layout.astro";
import Home from "../../components/home/Home.astro";
---

<Layout title="auxolotl.org">
	<Home />
</Layout>
```

nyawox commented:
I initially went with Nixエコシステムに代わる新たな選択肢 because I felt it describes better, but if I think now Nixエコシステムの代替 fits better, since it's below the title

AngryAnt commented:
Whoop! I'm in the middle of breakfast, but just noticed this and figured I'd drop what notes I could on the comments here:
- Without remembering context, ad-hoc also communicates "bit by bit", "continuously changing" on top of "interim". I don't recall specific JP phrases I've come across covering that, but I'm thinking there's probably a good blend available?
- I don't have experience with tech JP, but at least in tech DK there are lots of words which do technically have full translations, but the EN words are what is used day to day. So personally I would think going with katakana for broadly understood concepts like "community" in the online sense makes sense? Perhaps "ecosystem" as you use it is a better example.
  - Not suggesting a change - just replying to your point about katakana use which from my perspective seems quite natural.
- For "education" there's no need to go formally specific - "learning" is an excellent substitute if that helps you get closer to a well-fitting word.
- For "logistics", think hands-on/practical/concrete - as opposed to committee being more of a direction-setting exercise.

For the rest I should probably read the actual context when I have some time at a computer :)

Very nice work! I did not find nearly the time on my end - currently sitting at about 50% completion ;)

nyawox commented:
Thank you for writing it up. I'll take some time to digest and make adjustments to my translation when I have the energy. Thanks for your effort!

nyawox commented:
OK, I've made some adjustments and rebased onto the latest main branch. I'm not confident enough, and I think it's far from perfect, but I believe it's alright.
Far better than something like Google translator:)

nyawox commented:
Rebased once again to resolve conflicts. I had someone outside the community take a glance, and they said it's likely okay; I just had to fix a blatant typo :)
But not sure who can merge now that apparently I'm the only native Japanese speaker here.
